src/renderer/Dataframe.js

```diff
@@ -498,7 +499,9 @@ export default class Dataframe extends DummyDataframe {
                     if (metadata.properties[propertyName].type === 'category') {
                         return metadata.IDToCategory.get(this._dataframe.properties[propertyName][index]);
                     } else {
-                        return this._dataframe.properties[propertyName][index];
+                        return this._dataframe.properties[propertyName][index] <= Number.MIN_SAFE_INTEGER
```

Comment:
It's a very good idea to put it here, but I would expect a check like `!==  Number.MIN_SAFE_INTEGER` not with `<=`